pyracf/common/segment_trait_error.py

```diff
@@ -0,0 +1,16 @@
+"""Exception to use when the user passes bad segment-trait name(s) on an add/alter request."""
+
+
+class SegmentTraitError(Exception):
+    """
+    Raised when a user passes a bad segment-trait combination in the traits dictionary.
+    """
+
+    def __init__(self, bad_traits: list, profile_type: str) -> None:
+        self.message = "Unable to build Security Request.\n\n"
+        for trait in bad_traits:
+            self.message += f"'{trait}' is not a known segment-trait "
+            self.message += f"combination for '{profile_type}'.\n"
```

Comment:
Change to:

```python
            self.message += (
                f"'{trait}' is not a known segment-trait "
                + f"combination for '{profile_type}'.\n"
            )
```

Reply:
Thank you! For the life of me, I was drawing a blank on how to structure that. This is perfect.